### PR TITLE
docs: describe how we do code reviews

### PR DESCRIPTION
**Pull request by martinvonz**
Let me know if you disagree with the text

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have added myself to the contributors in `CHANGELOG.md` (optional)

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -25,6 +25,31 @@ use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+Unlike most GitHub projects (but like most VCS projects), we care more about the
+contents of commits than about the contents of PRs. We review each commit
```
Comment (yuja) [inline docs/contributing.md:29]: nit: `s/most/many/` GitHub projects? I don't have statistics, but "most" seems too strong.

File: docs/contributing.md [hunk 2]
```diff
@@ -25,6 +25,31 @@ use GitHub pull requests for this purpose. Consult
...
+
+Each commit should ideally do one thing. For example, if you need to refactor a
+function in order to add a new feature cleanly, put the refactoring in one
+commit and the new feature in a different commit. If the refactoring itself
+consists of many parts, try to separate out those into separate commits. You can
+use `jj split` to do it if you didn't realize ahead of time how it should be
+split up. Include tests and documentation in the same commit as the code the
+test and document. The commit message should describe the changes in the commit;
+the PR description can even be empty, unless you
+
```
Comment (yuja) [inline docs/contributing.md:40]: missed conditional?

**Review by yuja — APPROVED** [review]

File: docs/contributing.md (hunk 1 above)
Comment (martinvonz) [reply docs/contributing.md:29]: I'd be very surprised if it's not most, but there's no need to be so specific anyway, so I changed to "many" here. I also changed from "most VCS project" to "many VCS project".

File: docs/contributing.md (hunk 2 above)
Comment (martinvonz) [reply docs/contributing.md:41]: Oops, fixed.

**Pull request merged**